docs/docs/about/modelling/setup/facility_inputs/pump_modelling/pump_charts.md

```diff
@@ -76,7 +76,7 @@ FACILITY_INPUTS:
     TYPE: PUMP_CHART_VARIABLE_SPEED
     UNITS:
         RATE: <rate unit, currently only AM3_PER_HOUR supported>
-        HEAD: <polytropic head unit, M, KJ_PER_KG, JOULE_PER_KG supported>
+        HEAD: <M supported>
```

Comment:
```suggestion
        HEAD: <polytropic head unit, only M supported>
```

Reply:
Thanks. It has been updated as Ole Peter suggested, 
HEAD: <polytropic head unit, only M supported>

Reply:
> Thanks. It has been updated as Ole Peter suggested, HEAD: <polytropic head unit, only M supported>

I think you have forgotten to push the update to here, just done it locally.

Reply:
Thanks, Ole Peter. I had it updated again. Could you please to take a review if it was updated / pushed in the remote repository? The new updated is below.